City-Bureau/city-scrapers-indianapolis
# dynamically scrape year  for indygo org.

@SimmonsRitchie 

## What's this PR do?
Edit the committees for Indygo to dynamically retrieve the current year. 

## Why are we doing this? 
Previously, these committees had the year '2023' hardcoded into the date. Better if the entire date is dynamically scraped.

## Steps to manually test
Reviewer can check the source webpage to check that the dates are correctly scraped. Reviewer should also run pytest. 

## Are there any smells or added technical debt to note? 
The pytest is not passing for any of the scrapers. I updated the html files for each of the organizations, but the startdate and meeting ids are still coming back as failed.
